crates/chia-consensus/src/lib.rs

```diff
@@ -6,5 +6,8 @@ pub mod error;
 pub mod fast_forward;
 pub mod gen;
 pub mod generator_rom;
+pub mod generator_types;
 pub mod merkle_set;
 pub mod merkle_tree;
+pub mod multiprocess_validation;
```

Comment:
Imo we should name this something more clear, like `spend_bundle_validation`. We aren't using multi-processing anymore.

Reply:
Done